avro2tf/src/main/scala/com/linkedin/avro2tf/parsers/Avro2TFJobParamsParser.scala

```diff
@@ -339,6 +341,23 @@ object Avro2TFJobParamsParser {
         """Optional.
           |Whether to pass through inputs and only change outputs formats, num partitions, etc""".stripMargin
       )
+
+    opt[Seq[String]](Avro2TFJobParamNames.FEATURE_LIST_CAP)
```

Comment:
It might be simpler for the user to specify this option in the config file where the tensor is defined, instead of separately in a command-line parameter. For example, I have been using the shared feature lists param (also a command-line argument), and have noticed it is easy to accidentally change the command-line without changing the config, and vice versa. Overall I think the more configuration is in the json config file the easier it is for the user.

Reply:
Thanks for the suggestion. I agree that will make the config/parameters less error-prone. However, internally, the avro2tf config is not provided by user but inferred from Quasar, putting the new param to config for now will make this feature not available to PC pipelines. After we have a good way to propagate these configs from quasar to PC, we can then move them to the avro2tf config.

Reply:
let's do json config. I can put a parameter in trainable to populate from Quasar. Does it sound good?

Reply:
sure, let's proceed, given that you will work on PC side.